packages/desktop/src/components/channel-message.js

```diff
@@ -717,6 +718,42 @@ const MemberDisplayName = React.forwardRef(
   )
 );
 
+const AppDisplayName = React.forwardRef(
+  ({ displayName, color, ...props }, ref) => (
+    <div
+      ref={ref}
+      css={(theme) =>
+        css({
+          lineHeight: 1.2,
+          color: color ?? theme.colors.pink,
+          fontWeight: "500",
+          display: "flex",
+          alignItems: "center",
+        })
+      }
+      {...props}
+    >
+      {displayName}
+      <span
```

Comment:
stole the little App badge from slack:
<img width="483" alt="Screenshot 2022-06-04 at 15 43 08" src="https://user-images.githubusercontent.com/314617/172003333-8c1adbb3-a37b-4a1f-b8c8-1030a3ecdb96.png">